app/prj.conf

```diff
@@ -207,3 +208,21 @@ CONFIG_NPM1300_CHARGER=y
 # Task Watchdog
 CONFIG_TASK_WDT=y
 CONFIG_TASK_WDT_CHANNELS=6
+
+# Memfault
+CONFIG_MEMFAULT=y
+CONFIG_MEMFAULT_NCS_PROJECT_KEY="key"
+CONFIG_MEMFAULT_HTTP_PERIODIC_UPLOAD=y
+CONFIG_MEMFAULT_ROOT_CERT_STORAGE_NRF9160_MODEM=y
+CONFIG_MODEM_KEY_MGMT=y
+CONFIG_MEMFAULT_LOGGING_ENABLE=y
+
+# Periodic upload of metrics
+CONFIG_MEMFAULT_HTTP_ENABLE=y
+CONFIG_MEMFAULT_HTTP_PERIODIC_UPLOAD_INTERVAL_SECS=1800
```

Comment:
**suggestion:** align with cloud update interval.

```suggestion
CONFIG_MEMFAULT_HTTP_PERIODIC_UPLOAD_INTERVAL_SECS=3600
```

Reply:
Otherwise this violates the intended behaviour of the device to only send data to the cloud every hour.

Reply:
I think the metrics can be sent even at longer intervals. I.e 6 or 12 hours.